core/src/main/scala/org/apache/spark/h2o/converters/SupportedDataset.scala

```diff
@@ -41,15 +41,15 @@ object SupportedDataset {
     val typeTag = ttag
     override def toH2OFrame(sc: SQLContext, frameKeyName: Option[String]): H2OFrame = {
       val tpe = ttag.tpe
-      val constructorSymbol = tpe.declaration(nme.CONSTRUCTOR)
+      val constructorSymbol = tpe.declaration(nme.CONSTRUCTOR) // To be compatible with Scala 2.10
```

Comment:
Just a question, why is here the comment // To be compatible with Scala 2.10 ? Would this be done in different way in scala 2.11 ? 

I have no problem with this, I'm just don't fully understand this short piece of code.

Thanks!